### PR TITLE
add arrow buttons

### PR DESCRIPTION
**Pull request by jerminatorhits**
I'm getting a weird button lag / delay when a button is clicked

UPDATE: Ohh nvm. it lags when I use the mobile screen Dev tool on Chrome; shrinking the browser makes it work seamlessly!

UPDATE 2: deployed to gh-pages and issue reappeared. Will investigate starting with: https://www.sitepoint.com/5-ways-prevent-300ms-click-delay-mobile-devices/
and https://developers.google.com/web/updates/2013/12/300ms-tap-delay-gone-away

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,18 +6,80 @@
...
+        outline: none;
+      }
+      button:active {
+        background-color: #272727;
+      }
       #game {
-          width: 600px;
-          height: 600px;
+        width: 100vw;
+        height: 100vw;
```
Comment (killscreen) [inline index.html]: I think [`vmin`](https://developer.mozilla.org/en-US/docs/Web/CSS/length) would work here to keep the screen in frame regardless of aspect ratio, and might remove the need for the `@media` usages below

Comment (jerminatorhits) [reply index.html]: ah nice. changed

**Comment by jerminatorhits** [comment]
think this is 'functional' for now but could use some optimization. Will look into using 'touchend' click handler in the next PR as well as maybe hiding the buttons for desktop (might need to or might not). Also the game feels twice as fast on mobile lol

**Review by killscreen — APPROVED** [review]
lgtm!

**Pull request merged**